src/itdelatrisu/opsu/options/Options.java

```diff
@@ -399,15 +399,17 @@ public void selectItem(int index, GameContainer container) {
 
 			@Override
 			public String getValueString() {
-				return String.format((getTargetFPS() == 60) ? "%dfps (vsync)" : "%dfps", getTargetFPS());
+				return String.format((getTargetFPS() == -1) ? "Unlimited" : 
```

Comment:
Take `(getTargetFPS() == -1) ? "Unlimited" :` out of `String.format()`.

Reply:
Noted.